logutils/logutils.go

```diff
@@ -207,13 +257,12 @@ func (f *Formatter) Format(entry *log.Entry) ([]byte, error) {
 			stringifiedValue = stringer.String()
 		} else {
 			// No string method, use %#v to get a more thorough dump.
-			stringifiedValue = fmt.Sprintf("%#v", value)
+			fmt.Fprintf(b, " %v=%#v", key, value)
```

Comment:
On this branch, value doesn't have a String() method, I'm explicitly wanting the `%#v` format.